pages/shared/groupWorkInfoContainer.ejs

```diff
@@ -52,8 +52,12 @@
       </div>
       <% } %>
       <span id='group-member'><b>Group members: </b></span>
-      <% group_info.forEach(function(row) {%>
-      <li><%= row.user_name %> (<%= row.uid %>)</li>
+      <% groupMembers.forEach(function(user) {%>
+      <% if (hasRoles) { %>
+        <li><%= user.uid %> - <%= rolesInfo.roleAssignments[user.uid] ? rolesInfo.roleAssignments[user.uid].join(", ") : "None"%></li>
+      <% } else { %>
+        <li><%= user.uid %></li>
+      <% } %>
```

Comment:
I think having a `roleAssignments` property for each item in `groupMembers` would make more sense in this frontend code, but we're unsure on how to cleanly combine the `get_role_assignments` and `get_group_members` SQL queries. We thought it would be cleaner to keep `roleAssignments` as a separate object within `rolesInfo` in the case that a group assessment doesn't have roles.

If you would prefer we combine these before this gets merged though, let us know!